### PR TITLE
add analytics_label in FcmOptions

### PR DESCRIPTION
**Pull request by chemidy**
-add analytics_label in FcmOptions

#301

### CODE REVIEW

**Comment by stripathi669** [comment]
I came here to raise the same request. Thank you for that.

Any rough idea when can we see this in pypi release? In terms of (weeks or months etc)

**Comment by chemidy** [comment]
I have to add tests and fix some bug in my PR before :)

File: firebase_admin/_messaging_utils.py [hunk 1]
```diff
@@ -387,6 +409,27 @@ def __init__(self, title=None, subtitle=None, body=None, loc_key=None, loc_args=
...
+
+    Args:
+        analytics_label: contains additional options for features provided by the FCM iOS SDK
+            (optional).
+    """
+
+    def __init__(self, analytics_label=None):
+        self.analytics_label = analytics_label
+
+class FcmOptions(object):
```
Comment (hiranya911) [inline firebase_admin/_messaging_utils.py]: Add one more empty line.

File: firebase_admin/_messaging_utils.py [hunk 2]
```diff
@@ -387,6 +409,27 @@ def __init__(self, title=None, subtitle=None, body=None, loc_key=None, loc_args=
...
+    """Options for features provided by SDK.
+
+    Args:
+        analytics_label: contains additional options to use across all platforms (optional).
+    """
+
+    def __init__(self, analytics_label=None):
+        self.analytics_label = analytics_label
+
```
Comment (hiranya911) [inline firebase_admin/_messaging_utils.py]: One more empty line.

**Pull request closed without merging**